docs/pages/access-controls/access-graph/self-hosted-helm.mdx

```diff
@@ -1,21 +1,18 @@
 ---
-title: Run Teleport Access Graph on Self-Hosted Clusters with Helm
+title: Run Teleport Policy's Access Graph feature on Self-Hosted Clusters with Helm
 ---
 
-Using Access Graph with a self-hosted Teleport cluster requires
-setting up the Teleport Access Graph (TAG) service.
-TAG is a dedicated service which uses PostgreSQL as its backing storage
-and communicates with Auth Service and Proxy Service
+Using Teleport Policy's Access Graph with a self-hosted Teleport cluster requires
```

Comment:
This sentence feels awkward. "Using access graph requires setting up access graph" feels redundant.

Reply:
agreed, way too many 'access graphs'!